### PR TITLE
Added Raw Ops operation: Elu to Tensorflow Frontend

### LINKED ISSUE #9932: Elu

### PR DESCRIPTION
**Pull request by AdvancedMicroDev**
closes #9932

### CODE REVIEW

**Comment by AdvancedMicroDev** [comment]
Hello @Aarsh2001, waiting for you to review the changes & provide your feedback on the same :slightly_smiling_face:

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -2931,3 +2931,71 @@ def test_tensorflow_Softmax(
...
+# ):
+#     input_dtype, x = dtype_and_x
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         x=x[0],
+#     )
```
Comment (Aarsh2001) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:3001]: Hi @AdvancedMicroDev, can you please remove these comments. Thanks !

**Comment by Aarsh2001** [comment]
The PR looks good to go !

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py (hunk 1 above)
Comment (AdvancedMicroDev) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:3001]: Yes Sure :smile:

**Comment by AdvancedMicroDev** [comment]
@Aarsh2001 I have done the required changed, you may review the latest PR now... Thanks :slightly_smiling_face:

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py (hunk 1 above)
Comment (AdvancedMicroDev) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py]: @Aarsh2001 I have done the required changes, you may review the latest PR now... Thanks! :slightly_smiling_face:

**Review by Aarsh2001 — COMMENTED** [review]
LGTM

**Comment by AdvancedMicroDev** [comment]
Thanks a lot!

**Pull request merged**